### ISSUE TITLE
When a site is activated, keep track of it somewhere so agencies can request login access easily

### ISSUE THREAD

**Issue by zackkatz** [trustedlogin/client#45]
Re: tracy apps call (remember?)

![](https://i.gravityview.co/9OzuiK+)

**Comment by inztinkt** [comment]
@zackkatz since SaaS is now getting a decrypted site_url value in the envelope, should we make the change on there to not 'wipe' any non-active sites? That will give you a list of sites who have a) triggered the TL functionality and b) their TL status.

**Comment by zackkatz** [comment]
@inztinkt Yes, once we expand to agencies, it makes sense to not wipe. Until then, I think it's good to wipe for security purposes, just in case.

**Comment by zackkatz** [comment]
Superseded. Keeping track of activated sites so an agency can request access is now a SaaS surface: the Sites list, plus the Connected Installs view being built as [TL-22](https://linear.app/trustedlogin/issue/TL-22/saas-connected-installs-management-ui) (site URL, plugin version, last seen, revoke). The Client SDK is the wrong layer for a cross-site registry.
